ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3028,3 +3064,34 @@ SDBPC:
   - supply_air_damper_percentage_sensor
   implements:
   - MONITORING
+
+HWVM:
```

Comment:
This type is a subset of the fields of HWRC.  The only material difference is that feedback is required in this one.  Was this difference the reason to create this type?  

Also, they have the same description - need to differentiate them.

Reply:
adjusted description